server/src/model-manager.ts

```diff
@@ -299,7 +299,9 @@ export class ModelManager {
     if (await checkFileExists(fsPath)) {
       const stat = await fs.promises.stat(fsPath)
       if (stat.isFile()) { // Example: <image src=""/> resolves to 'modules/m123' which is a directory.
-        return await fs.promises.readFile(fsPath, 'utf-8')
+        return ['.jpg', '.png'].some((ext) => uri.endsWith(ext))
+          ? 'image'
```

Comment:
Maybe something easier to grep like `<fakeimagedata>`

```suggestion
          ? '<fakeimagedata>'
```